<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nordend</class>
 <widget class="QMainWindow" name="nordend">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Verdana&quot;;
</string>
    </property>
    <property name="text">
     <string>Serial port status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="serialport_status">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/assets/refresh.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="open_serial">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(30, 30, 30);
font: 9pt &quot;Verdana&quot;;
border-radius: 10px;
background: white;
}
QPushButton:hover {
font: 700 9pt &quot;Verdana&quot;;
}
QPushButton:pressed {
font: 700 9pt &quot;Verdana&quot;;
color: white;
background: transparent;
border: 3px  solid white;
}

</string>
    </property>
    <property name="text">
     <string>Open Serial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_serial">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
/*font: 500 11pt &quot;Calibri&quot;;*/
font: 9pt &quot;Verdana&quot;;
border-radius: 10px;
border: 2px  solid white; /*rgb(255, 38, 114);*/
}
QPushButton:hover {
font: 700 9pt &quot;Verdana&quot;;
}
QPushButton:pressed {
font: 700 9pt &quot;Verdana&quot;;
background: white;
color: black;
}
</string>
    </property>
    <property name="text">
     <string>Close Serial</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Verdana&quot;;
</string>
    </property>
    <property name="text">
     <string>Serial port detected:</string>
    </property>
   </widget>
   <widget class="QLabel" name="serial_port_detected_name">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Verdana&quot;;
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="avionics_box">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>221</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Data AV</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="2" column="1">
      <widget class="QLabel" name="pressure">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="altitude_sensor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Sea level P.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="temperature">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="humidity_label">
       <property name="text">
        <string>Humidity</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="hpa_unit_label">
       <property name="text">
        <string>hPa</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="m_s_unit_label">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="m_unit_label_3">
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pressure_label">
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="temperature_label">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>hPa</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QGroupBox" name="avionics_box_3">
       <property name="title">
        <string>Miaou</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="4" column="2">
         <widget class="QLabel" name="m_unit_label_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="humidity_label_3">
          <property name="text">
           <string>RSSI</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="celcius_label_3">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="hpa_unit_label_3">
          <property name="text">
           <string>kHz</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="humidity_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="altitude_label_telemetry_3">
          <property name="text">
           <string>Vibe check</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="pressure_label_3">
          <property name="text">
           <string>Freq. err.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="pressure_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="temperature_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="m_s_unit_label_3">
          <property name="text">
           <string>dBm</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="temperature_label_3">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>SNR</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="time_since_last_Rx">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Last packet's timestamp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="sea_level_pressure_edit">
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(144, 144, 144);</string>
       </property>
       <property name="text">
        <string>1013.25</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="altitude_label_telemetry">
       <property name="text">
        <string>Altitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="humidity">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="celcius_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;o&lt;/span&gt;C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>-10</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: white;
	font: 16pt &quot;Calibri Light&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;NORDEND GS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>39</y>
      <width>721</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Propulsion</string>
    </property>
    <widget class="QPushButton" name="prop_diagram">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>671</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
qproperty-icon: url(:/assets/Prop_background_V1.svg);
background: transparent;
qproperty-iconSize: 700px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="vent_GSE">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>340</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	background: transparent;
}
QCheckBox::indicator {
     width: 50px; 
     height: 50px; 
}

QCheckBox::indicator:partiallychecked {
	image: url(:/assets/GS_valve_V2_Unknown.svg);
}

QCheckBox::indicator:checked {
    image: url(:/assets/GS_valve_V2_OpenV.svg);
}

QCheckBox::indicator:unchecked {
	image: url(:/assets/GS_valve_V2_CloseH.svg);
}

QCheckBox::indicator:hover {
	background-color: rgba(0, 0, 0, 50); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>330</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	background: transparent;
}
QCheckBox::indicator {
     width: 50px; 
     height: 50px; 
}

QCheckBox::indicator:checked {
	image: url(:/assets/GS_valve_V1_OpenV.png);
}

QCheckBox::indicator:unchecked {
	image: url(:/assets/GS_valve_V1_CloseH.png);
}

QCheckBox::indicator:hover {
	background-color: rgba(0, 0, 0, 50); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>330</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	background: transparent;
}
QCheckBox::indicator {
     width: 50px; 
     height: 50px; 
}

QCheckBox::indicator:checked {
	image: url(:/assets/GS_valve_V1_OpenV.png);
}

QCheckBox::indicator:unchecked {
	image: url(:/assets/GS_valve_V1_CloseH.png);
}

QCheckBox::indicator:hover {
	background-color: rgba(0, 0, 0, 50); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_7">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	background: transparent;
}
QCheckBox::indicator {
     width: 50px; 
     height: 50px; 
}

QCheckBox::indicator:checked {
	image: url(:/assets/GS_valve_V1_OpenV.png);
}

QCheckBox::indicator:unchecked {
	image: url(:/assets/GS_valve_V1_CloseH.png);
}

QCheckBox::indicator:hover {
	background-color: rgba(0, 0, 0, 50); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_8">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>148</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	background: transparent;
}
QCheckBox::indicator {
     width: 50px; 
     height: 50px; 
}

QCheckBox::indicator:checked {
	image: url(:/assets/GS_valve_V1_OpenV.png);
}

QCheckBox::indicator:unchecked {
	image: url(:/assets/GS_valve_V1_CloseH.png);
}

QCheckBox::indicator:hover {
	background-color: rgba(0, 0, 0, 50); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_9">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>148</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	background: transparent;
}
QCheckBox::indicator {
     width: 50px; 
     height: 50px; 
}

QCheckBox::indicator:checked {
	image: url(:/assets/GS_valve_V1_OpenV.png);
}

QCheckBox::indicator:unchecked {
	image: url(:/assets/GS_valve_V1_CloseH.png);
}

QCheckBox::indicator:hover {
	background-color: rgba(0, 0, 0, 50); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="fill_GSE">
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>230</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	background: transparent;
}
QCheckBox::indicator {
     width: 50px; 
     height: 50px; 
}

QCheckBox::indicator:partiallychecked {
	image: url(:/assets/GS_valve_V2_Unknown.svg);
}

QCheckBox::indicator:checked {
    image: url(:/assets/GS_valve_V2_OpenV.svg);
}

QCheckBox::indicator:unchecked {
	image: url(:/assets/GS_valve_V2_CloseH.svg);
}

QCheckBox::indicator:hover {
	background-color: rgba(0, 0, 0, 50); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_valves">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>560</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(30, 30, 30);
font: 9pt &quot;Verdana&quot;;
border-radius: 10px;
background: white;
}
QPushButton:hover {
font: 700 9pt &quot;Verdana&quot;;
}
QPushButton:pressed {
font: 700 9pt &quot;Verdana&quot;;
color: white;
background: transparent;
border: 3px  solid white;
}

</string>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
    <widget class="QPushButton" name="valve_test">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>340</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background: transparent;
	qproperty-icon: url(:/assets/GS_valve_V2_CloseH.svg);
   qproperty-iconSize: 50px;
}
QPushButton:hover {
	background-color: rgba(0, 0, 0, 50); 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>721</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Command</string>
    </property>
    <widget class="QPushButton" name="abort_cmd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font: 9pt &quot;Verdana&quot;;
border-radius: 10px;
background: red;
}
QPushButton:hover {
font: 700 9pt &quot;Verdana&quot;;
}
QPushButton:pressed {
font: 700 9pt &quot;Verdana&quot;;
color: white;
background: transparent;
border: 3px  solid red;
}</string>
     </property>
     <property name="text">
      <string>ABORT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ignition_cmd">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font: 9pt &quot;Verdana&quot;;
border-radius: 10px;
background: red;
}
QPushButton:hover {
font: 700 9pt &quot;Verdana&quot;;
}
QPushButton:pressed {
font: 700 9pt &quot;Verdana&quot;;
color: white;
background: transparent;
border: 3px  solid red;
}</string>
     </property>
     <property name="text">
      <string>IGNITION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnect_cmd">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font: 9pt &quot;Verdana&quot;;
border-radius: 10px;
background: red;
}
QPushButton:hover {
font: 700 9pt &quot;Verdana&quot;;
}
QPushButton:pressed {
font: 700 9pt &quot;Verdana&quot;;
color: white;
background: transparent;
border: 3px  solid red;
}</string>
     </property>
     <property name="text">
      <string>DISCONNECT</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>label_11</zorder>
   <zorder>serialport_status</zorder>
   <zorder>open_serial</zorder>
   <zorder>close_serial</zorder>
   <zorder>label_13</zorder>
   <zorder>serial_port_detected_name</zorder>
   <zorder>avionics_box</zorder>
   <zorder>label_39</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
